scripts/manpage_generation.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+# Directory to store the generated manpages
+MANPAGE_DIR="../man"
+
+# Generate the manpage using argparse-manpage
+PYTHONPATH=. argparse-manpage --pyfile man/__init__.py --function get_parser --manual-title="General Commands Manual" --description="Automates the conversion of Red Hat Enterprise Linux derivative distributions to Red Hat Enterprise Linux." --project-name "Convert2RHEL" --prog="convert2rhel" > "$MANPAGE_DIR/convert2rhel.8"
```

Comment:
One important note here... In the `--project-name` we are hard-coding the name `Convert2RHEL`, but to apply with the old manpage and keep the consistency, I believe we should use `convert2rhel major.minor.patch`, where `major.minor.patch` probably needs to be read from some other file that contains that information.

Reply:
gotcha I will definitely get that fixed in the command

Reply:
FYI we use `major.minor.micro` to not be confused with RPM patches 😛

Reply:
Oh, that's correct. `major.minor.micro`. Thought it was `patch`.